charts/internal/machine-controller-manager/shoot/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Normally you would have in this chart also a role and a rolebinding to manipulate secrets in the shoot namespace. I don't see them here?